lib/iris/cube.py

```diff
@@ -3055,7 +2983,32 @@ def _order(array):
 
         return cube_xml_element
 
-    def copy(self, data=None):
+    def replace(self, data, dtype=None, fill_value=None):
```

Comment:
@pp-mo I've added a note to the **Open Questions** in the [DataManager Integration](https://github.com/SciTools/iris/projects/5) project, so that we don't forget :wink: